src/index.ts

```diff
@@ -163,3 +163,4 @@ export {
   ChildrenComponentProps,
   ContentComponentProps,
 } from './lib'
+export { ShorthandRenderer } from '../types/utils'
```

Comment:
yes, returning to your comment about how types are arranged on files - we should definitely consider some grooming there